recipes/pepnovo/meta.yaml

```diff
@@ -0,0 +1,29 @@
+about:
+  home: http://proteomics.ucsd.edu/Software/PepNovo/
+  license_file: LICENSE
+  summary: PepNovo serves as a high throughput de novo peptide sequencing tool for tandem mass spectrometry data
+
+package:
+  name: pepnovo
+  version: 20101117
+
+build:
+  number: 0
+  skip: False
+
+source:
+  fn: PepNovo.20120423.zip
+  url: http://proteomics.ucsd.edu/Software/PepNovo/PepNovo.20120423.zip
+  md5: af2a863487cb53bf0e92f00d395c4f0d
+
+requirements:
+  build:
+    - gcc
```

Comment:
Please include llvm for osx, like described here: https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md#cc